README.md

```diff
@@ -23,10 +23,15 @@ As a highly common scenario for interfaces between WebIDL and Wasm, this should
 
 ## Algorithm
 
-The validation algorithm is effectively the standard UTF-16 validation algorithm, iterating through the string and pairing UTF-16 surrogates, failing validation for any unpaired surrogates
-or invalid surrogate prefix codes.
+The validation algorithm is effectively the standard UTF-16 validation algorithm, iterating through the string and pairing UTF-16 surrogates, failing validation for any unpaired surrogates or invalid surrogate prefix codes.
```

Comment:
What does "invalid surrogate prefix codes" mean?

Reply:
How does the following sound:

```suggestion
The validation algorithm is effectively the standard UTF-16 validation algorithm, iterating through the string and pairing UTF-16 surrogates, failing validation for any unpaired surrogates.
```

Reply:
Much clearer, thanks!